other/lru_cache.py

```diff
@@ -0,0 +1,127 @@
+class Double_Linked_List_Node():
```

Comment:
Do not use `snake_cases` for class naming. Also, no `()` needed as you are not subclassing from any class.

You might wanna try
```
class DoubleLinkedListNode:
```